Src/Orbiter/Body.h

```diff
@@ -163,39 +163,28 @@ class Body {
 	inline const Vector &Acceleration() const { return acc; };
 
 protected:
-	double mass;         // current body mass [kg]
-	double size;         // (mean) body radius [m]
-	Vector albedo;       // object albedo (RGB, 0-1)
-	double vislimit;     // total visibility limit (in units of viewport vertical)
-	double spotlimit;    // spot visibility limit (in units of viewport vertical)
+	double mass { 0.0 };         // current body mass [kg]
+	double size { 0.0 };         // (mean) body radius [m]
+	Vector albedo { 1, 1, 1 };   // object albedo (RGB, 0-1)
+	double vislimit { 1e-3 };    // total visibility limit (in units of viewport vertical)
+	double spotlimit { 1e-3 };   // spot visibility limit (in units of viewport vertical)
 
 	Vector acc;          // current acceleration vector
 
-	VISHANDLE hVis;      // visual identifier passed to messages (NULL=no visual)
+	VISHANDLE hVis { nullptr };      // visual identifier passed to messages (NULL=no visual)
 
-	const CelestialBody *cbody; // orbit reference body
+	const CelestialBody* cbody { nullptr }; // orbit reference body
 
-	void Setup ();       // initialise body with default params
+	std::string name { "" };          // object name
+	std::string filename { "" };
```

Comment:
IMHO this should be an instance of [`fs::path`](https://en.cppreference.com/w/cpp/filesystem/path).

Reply:
Also, default constructor for `std::string` will create an [empty string](https://en.cppreference.com/w/cpp/string/basic_string/basic_string), so there is no need to add `{ "" }` afterwards.

Reply:
And, it would be nice IMHO to suffix internal (private and protected) member variables with `_` to: (a) identify that they are internal, and (b) if you later want to expose it, you don't have to come up with a creative function name, such as `getName()` or something, and instead you can just add a function called `name()`.